constants/ContributorList.json

```diff
@@ -12,6 +12,7 @@
     "Empa_",
     "Erymanthus",
     "Frost19",
+    "Gillsplash",
```

Comment:
> This is deprecated use Contributors.json instead.

No more contributors should be added to this json

Reply:
oh alr